test/ref_ops_nonstd_shape_test.cpp

```diff
@@ -55,4 +55,71 @@ TEST_CASE(argmin_test_nonstd_shape)
     EXPECT(migraphx::verify_range(result_vec, res_gold_vec));
 }
 
+TEST_CASE(squeeze_transpose_test)
+{
+    migraphx::program p;
+    auto* mm                = p.get_main_module();
+    std::vector<float> data = {
+        1.2255,  1.6834,  -2.0305, -0.3221, 0.4701,  0.2583, 0.7545,  2.5758,  -1.6849,
+        0.0928,  0.9022,  -0.8765, -0.4090, 0.9301,  2.0724, -1.5706, 0.4867,  -0.1493,
+        0.6957,  -0.2179, 0.7142,  0.7177,  0.0183,  1.3497, 1.2255,  1.6834,  -2.0305,
+        -0.3221, 1.2255,  1.6834,  -2.0305, -0.3221, 1.2255, 1.6834,  -2.0305, -0.3221};
+    migraphx::shape s1{migraphx::shape::float_type, {4, 1, 3, 1, 3}};
+    auto l0 = mm->add_literal(migraphx::literal{s1, data});
+    auto l0_trans =
+        mm->add_instruction(migraphx::make_op("transpose", {{"permutation", {1, 2, 3, 0, 4}}}), l0);
+    mm->add_instruction(migraphx::make_op("squeeze"), l0_trans);
+    auto p_uncompiled = p;
+    p.compile(migraphx::ref::target{});
+    auto result          = p.eval({}).back();
+    auto expected_result = p_uncompiled.eval({}).back();
+    // contiguous is required to read the values in standard shaped order
+    auto tr_op               = migraphx::make_op("contiguous");
+    auto std_expected_result = tr_op.compute(result.get_shape(), {expected_result});
+    EXPECT(result == std_expected_result);
```

Comment:
Can you add check for the shape so we can see what the final shape should be in the test? Something like:

```cpp
EXPECT(result.get_shape() == migraphx::shape{migraphx::shape::float_type, {4, 3, 3}})`
```

Reply:
Done.